tests/integration/cards.test.js

```diff
@@ -139,6 +139,17 @@ describe('CARDS ROUTES - PUT /cards/{_id}', () => {
       expect(response.statusCode).toBe(400);
     });
 
+    it('should return a 403 when provide a useless areAnswersCommutative field on a non-FillTheGap card', async () => {
```

Comment:
Il faudrait aussi ajouter areAnswersCommutative au payload pour la carte FillTheGap dans le test `should update a ${card.template} card`

Reply:
done